webapp/channels/src/packages/mattermost-redux/src/selectors/entities/channels.ts

```diff
@@ -684,16 +684,12 @@ export const getTeamsUnreadStatuses: (state: GlobalState) => [Set<Team['id']>, M
     getAllChannels,
     getMyChannelMemberships,
     getChannelMessageCounts,
-    getUsers,
-    getCurrentUserId,
     isCollapsedThreadsEnabled,
     getThreadCounts,
     (
         channels,
         channelMemberships,
         channelMessageCounts,
-        users,
```

Comment:
Great catch. I know we have ESLint configured to not report unused parameters when later parameters are used, but I wonder if there's some way to have it report those for selectors in particular